client/src/components/QueryComponent/QueryComponent.scss

```diff
@@ -303,15 +303,34 @@
 
 .qc-monaco-layer {
   width: calc(100% - 535px);
-  margin-top: 40px;
+  margin-top: 42px;
   margin-left: 225px;
   position: fixed;
   float: left;
   z-index: 10;
   overflow: hidden;
-  background-color: transparent;
+  background-color: var(--input-color);
+  border-color:rgb(208, 208, 208);
+  border-width:1px;
+  border-style: solid;
+  box-shadow: none;
+  box-sizing:border-box;
+  padding: 0px;
 }
 
+.monaco-editor-background {
+  background-color: var(--input-color)  !important;
```

Comment:
I would really prefer that we didn't use `!important`. Could we make the CSS selector more specific so we can override properly? Additional classes in the selector will make it more specific and help the style "win out".

Reply:
Done